app/components/case-studies-components/CaseStudiesTitleText.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
```

Comment:
Try to remember to remove all unnecessary imports.

Reply:
aaaaaand it's gone